README.rst

```diff
@@ -348,7 +362,7 @@ The mandatory arguments are ``config``, ``signing-key``, ``protocol-version-majo
 
 The remaining arguments are optional parameters you want to update in your update proposal.
 
-You can also check your proposal's validity using the `Validate cbor files`_ command.
+You can also check your proposal's validity using the `validate-cbor` command. See: `Validate cbor files`_.
```

Comment:
```suggestion
You can also check your proposal's validity using the `validate-cbor` command. See: `Validate CBOR files`_.
```

Maybe we should also change the title for that section of the document to reflect this too.